Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -52,22 +74,46 @@ export function findAliasIdSql(documentId: string): string {
  * @param documentId The identifier of the document to retrieve
  * @returns SQL query string to retrieve a document
  */
-export function findDocumentByIdSql(documentId: string): string {
+export function findDocumentByIdSql(documentId: MeadowlarkId): string {
   return format(
     `
-    SELECT document_id, document_identity, edfi_doc
+    SELECT document_uuid, document_id, document_identity, edfi_doc
        FROM meadowlark.documents
        WHERE document_id = %L;`,
```

Comment:
```suggestion
       WHERE document_uuid = %L;`,
```

Reply:
That was a bad suggestion. The problem is not this function, it is that the repository is calling this one instead of the next function when doing a GET by ID query.